### ISSUE TITLE
Bilateral apical fibrosis

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6570]
**Preferred term label:** Bilateral apical fibrosis

**Synonyms**

**Definition (free text, please give PubMed ID)**
It is a bilateral reticular pattern of linear or lineonodular densities in apical portions of the lungs seen initially on HRCT and in case of progression also on standard chest x-ray. Bilateral apical fibrosis is a characteristic finding in ankylosing spondylitis.
PMID: 16376803

**Parent term (use hpo.jax.org/app)**
Pulmonary fibrosis HP:0002206

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Ankylosing spondylitis 

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
